Makefile

```diff
@@ -36,6 +40,13 @@ $(OBJ): $(GEN)
 %.h: %.xml
 	$(WAYLAND_SCANNER) client-header < $< > $@
 
+primary-selection-unstable-v1.xml:
+	cp $(WAYLAND_PROTOCOLS_DIR)/unstable/primary-selection/primary-selection-unstable-v1.xml ./
+xdg-decoration-unstable-v1.xml:
+	cp $(WAYLAND_PROTOCOLS_DIR)/unstable/xdg-decoration/xdg-decoration-unstable-v1.xml ./
+xdg-shell.xml:
+	cp $(WAYLAND_PROTOCOLS_DIR)/stable/xdg-shell/xdg-shell.xml ./
+
```

Comment:
If they are copied here like this they need to be removed also by the clean target. What's the benefit of doing it this way?

Reply:
When I cloned the repo, `make` failed due to protocol files not found. this makes sure that it is copied from `/usr/share/wayland-protocols` to current directory.
This way makes cross distro (ex: debian, arch) builds more easy for package maintainers.
I might be wrong.

Reply:
Hm that's no good, why did make not find the files? VPATH should tell make to go look in those directories for source files. Is this way maybe portable to non-gnu make, are you using a bsd make or something?
Could you share the whole error output when it fails?
Why do you think it makes a difference for cross distro builds? Don't both methods require the files to be in the same place.